Source/WebCore/page/Quirks.cpp

```diff
@@ -1191,7 +1215,6 @@ Quirks::StorageAccessResult Quirks::triggerOptionalStorageAccessQuirk(Element& e
     static NeverDestroyed<HashSet<RegistrableDomain>> kinjaQuirks = [] {
         HashSet<RegistrableDomain> set;
         set.add(RegistrableDomain::uncheckedCreateFromRegistrableDomainString("avclub.com"_s));
-        set.add(RegistrableDomain::uncheckedCreateFromRegistrableDomainString("gizmodo.com"_s));
         set.add(RegistrableDomain::uncheckedCreateFromRegistrableDomainString("deadspin.com"_s));
         set.add(RegistrableDomain::uncheckedCreateFromRegistrableDomainString("jalopnik.com"_s));
         set.add(RegistrableDomain::uncheckedCreateFromRegistrableDomainString("jezebel.com"_s));
```

Comment:
All these sites are part of the kinja network? 
They may exhibit the same issue?
Was it checked?

Reply:
Though Jezebel has been abandoned and taken by another group.

Let's see. Same network than Gizmodo.

* https://www.avclub.com
* https://deadspin.com
* https://gizmodo.com
* https://jalopnik.com
* https://kotaku.com
* https://qz.com
* https://www.theroot.com
* https://thetakeout.com
* https://www.theonion.com
* https://theinventory.com

Interesting clicking the Jezebel login button still open the kinja network of sites.
https://jezebel.com
![Screenshot 2023-12-15 at 16 04 01](https://github.com/WebKit/WebKit/assets/505230/28199cbc-4ef1-4ca5-a92e-24b592a757f6)

Reply:
This radar is about gizmodo.com, so I limit the change to gizmodo.
As you said, other sites need to be evaluated case by case, i.e. whether the old quirk still works, whether they belong to the same organization nowadays, whether they have different UI elements, etc, so I don't think they belong to the same change. We could do in a followup.

Reply:
We have expressed in multiple standards conversations that the limit of domains under which a same-party SSO domain gets cookie access should be 5.

Reply:
Do you suggest a change here? This is not part of the patch and it's added in https://trac.webkit.org/changeset/263383/webkit for the old quirk, so I don't know much about it.